packages/data-context/src/DataContext.ts

```diff
@@ -181,6 +181,7 @@ export class DataContext {
   @cached
   get cloud () {
     return new CloudDataSource({
+      // @ts-ignore
```

Comment:
To address error below. Open to better ideas here. https://app.circleci.com/pipelines/github/cypress-io/cypress/59948/workflows/efa9b4b3-12ca-4f83-a8d7-d5c9186f28d4/jobs/2493976
![Screenshot 2024-02-21 at 10 51 30 AM](https://github.com/cypress-io/cypress/assets/1271364/f012ee99-d367-4291-b5af-2a4e30e62fa1)